tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -238,18 +238,19 @@ stateMachine params = iterate (\s -> stateFor s.next) CLIENT_START_STATE
         CLIENT_START_STATE = state 1 [
             action clientHello client True,
             action changeCipherSpec client (params.zero_rtt /\ compatMode client)
-        ] (if params.retry then HELLO_RETRY_STATE else SERVER_HELLO_STATE)
+        ] (if (params.retry == 1) then HELLO_RETRY_STATE else SERVER_HELLO_STATE)
 
         HELLO_RETRY_STATE = state 2 [
-            action helloRetryRequest server True,
+            action serverHello server True,
             action changeCipherSpec server (compatMode server),
-            action changeCipherSpec client (~params.zero_rtt /\ compatMode client),
-            action clientHello client True
-        ] SERVER_HELLO_STATE
+            action changeCipherSpec server (~params.zero_rtt /\ compatMode server),
+            action clientHello client True,
+            action changeCipherSpec client (params.zero_rtt /\ compatMode client)
```

Comment:
same as lindsay's comment above on the ordering of clientHello and changeCipherSpec here